### PR TITLE
Allow excluding files to lint for single runner

### PR DESCRIPTION
**Pull request by astrauka**
@mmozuras Allow excluding files to lint for single runner

```
eslint:
  exclude:
    - 'app/assets/**/*'
```

### CODE REVIEW

File: lib/pronto/version.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Pronto
   module Version
-    STRING = '0.7.1'.freeze
+    STRING = '0.7.2'.freeze
```
Comment (astrauka) [inline lib/pronto/version.rb]: eager to lift the version to `0.8.0` but this requires pronto-rubocop gem update :/

**Comment by astrauka** [comment]
@mmozuras any feedback?

**Comment by doomspork** [comment]
This functionality would be tremendously useful with how we use Pronto.

**Comment by mmozuras** [comment]
@astrauka looks great, thanks! 🙇

**Comment by doomspork** [comment]
Thank you @astrauka and @mmozuras!

**Pull request merged**